e2e/tests/omnibus-sepolia.toml/BFP_Market_ETH.e2e.js

```diff
@@ -420,4 +420,130 @@ describe(require('path').basename(__filename, '.e2e.js'), function () {
+      args: [accountId, marketId, collateralAddress, ethers.utils.parseEther(String(500))],
+    });
+
+    const newDigest = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getAccountDigest',
+      args: [accountId, marketId],
+    });
+    log({ newDigest });
+
+    const newDepositedWeth = newDigest.depositedCollaterals.find(
+      (c) => c.collateralAddress === collateralAddress
+    );
+    log({ newDepositedWeth });
+    const { now, blockTimestamp } = getTimes(provider);
+    const buffer = 1; // 1s
+    const diff = now - blockTimestamp;
+
+    if (diff < 0) {
+      await wait(Math.abs(diff) + buffer * 1000);
+    }
+    await commitBfpOrder({
+      wallet,
+      accountId,
+      marketId,
+      sizeDelta: -0.01,
+    });
+    await wait(1000);
+    await wait(5000);
+
+    const newPosition = await settleBfpOrder({ wallet, accountId, marketId });
+
+    let marketConfiguration = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getMarketConfigurationById',
+      args: [marketId],
+    });
+
+    log({ newPosition, marketConfiguration });
+
+    // getMarginDigest
+    const marginDigest = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getMarginDigest',
+      args: [accountId, marketId],
+    });
+    log({ marginDigest });
+    //find out how much collateral is in USD
+    const collateralUsd = marginDigest.collateralUsd;
+    log({ collateralUsd });
+
+    log('Updating market configuration');
+    log('Previous minMarginUsd: ' + marketConfiguration.minMarginUsd);
+    // Create a new object with the updated minMarginUsd
+    const updatedMarketConfiguration = {
+      ...marketConfiguration,
+      minMarginUsd: collateralUsd + 1,
+    };
+    log('New minMarginUsd: ' + updatedMarketConfiguration.minMarginUsd);
+
+    const owner = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'owner',
+    });
+
+    await contractWrite({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'setMarketConfigurationById',
+      args: [
+        {
+          marketId,
+          ...updatedMarketConfiguration,
+        },
+      ],
+      impersonate: owner,
+    });
+
+    log('Market configuration updated');
+
+    const rewardDistributor = new ethers.Contract(
+      wethRewardsDistributor,
+      require('../../deployments/BfpRewardDistributor.json').abi,
+      wallet
+    );
+    const poolCollateralTypes = await rewardDistributor.getPoolCollateralTypes();
+    log({ poolCollateralTypes });
+
+    // liquidate position
+
+    const { tx, receipt } = await contractWrite({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'flagPosition',
+      args: [accountId, marketId],
+    });
+
+    log({ tx, receipt });
+    log('Flagged position for liquidation');
+
+    // Attempt to liquidate. This should complete successfully.
+    await contractWrite({
```

Comment:
There is a limit on how long each "it" is executed for. And this is the reason all other tests have many "it" sections (one per action". Please split this case. And add some asserts too as now only logs are there.

Reply:
I've split this in 2 parts and cleaned up unused bits

Reply:
Yeah thanks, looks ok

Reply:
![image](https://github.com/user-attachments/assets/476a0426-fbe5-47d4-b21c-ec68d4729715)

This is what I was referring to, takes over 20s.
Most likely due to waits. Maybe split a bit more and keep delays in a separate `it` case with telling caption?